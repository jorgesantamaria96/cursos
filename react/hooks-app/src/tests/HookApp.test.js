import { shallow } from "enzyme";
import { HooksApp } from "../HooksApp";

describe("Pruebas sobre el componente <Hookspp />", () => {
  test("debe renderizar el componente correctamente", () => {
    const wrapper = shallow(<HooksApp />);

    expect(wrapper).toMatchSnapshot();
  });
});
