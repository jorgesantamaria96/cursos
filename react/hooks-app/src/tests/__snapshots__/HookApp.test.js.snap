// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre el componente <Hookspp /> debe renderizar el componente correctamente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
